test/harness/asyncHelpers-throwsAsync-same-realm.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: |
+    Thenables that reject with instances of the realm specified constructor function
+    satisfy the assertion, without cross realms collisions.
```

Comment:
I was a bit confused by this description, it sounds like the opposite of what the test actually does.

Reply:
I copied it directly from assert-throws-same-realm.js. I *think* the description just needs to be updated in both, so I'll do that. But I'm not very familiar with realms yet.